src/services/handleTodoFilter.ts

```diff
@@ -0,0 +1,24 @@
+import { Todo } from '../types/Todo';
+
+export function handleTodoFilter(
+  todoFromServer: Todo[],
+  filterBy: string,
+  title = '',
+): Todo[] {
+  const filteredTodo = todoFromServer
+    .filter(todo => todo.title.toLowerCase().includes(title.toLowerCase()));
+
+  switch (filterBy) {
+    case 'all':
+      return filteredTodo;
```

Comment:
lines 12-13 and lines 21-23 have the same code and logic. You can use only last one.